public/tsconfig.json

```
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "typeRoots": [ "./types", "./node_modules/@types"],
```

Please follow the naming convention, rename `types` folder  to `typings`

```suggestion
    "typeRoots": [ "./typings", "./node_modules/@types"],
```